src/orca-loader/LoaderPaymentElement.res

```diff
@@ -19,26 +19,27 @@ let make = (componentType, options, setIframeRef, iframeRef, mountPostMessage) =
       setIframeRef(ref)
     }
 
+    let callbackFuncForExtractValFromDict = key => {
```

Comment:
Please rename this function to `callbackFuncForExtractingValFromDict ` and move it to Utils

Reply:
Updated